app/src/pages/components/FormComponent.jsx

```diff
@@ -15,16 +15,20 @@ const Container = styled.form`
     z-index: 100;
 `
 
-const Form = styled.div`
+const Form = styled.form`
     width: 40%;
     background: ${VARIABLES.darkgrey};
     padding: 40px;
 `
 
 function FormComponent(props) {
+    const submitHandler = event => {
+        event.preventDefault();

```

Comment:
would be nice to have smthing like that: 

```
    const submitHandler = event => {
        event.preventDefault();
        props.onSubmit && props.onSubmit(event)
```

Reply:
Done